### DISCUSSION TITLE
Problem to call scene turn_on

### DISCUSSION THREAD

**Discussion by Truble187** [custom-cards/button-card#799] [Q&A]
Hey guys,
if I want to call a scene turn on service, I get alway an error and I don't know why.
`tap_action:
  action: call-service
  service: scene.turn_on
  target:
    entity_id: scene.diningroom_scene2
`

Error:
![image](https://github.com/custom-cards/button-card/assets/79580773/2e478703-a4f9-4a4c-984c-d91a92e5cd50)

The scene works, if I call it over the developer setting.

**Comment by smartqasa** [comment]
You might try changing "target:" to "data:".
